src/System.Diagnostics.Process/src/System/Diagnostics/Process.Windows.cs

```diff
@@ -95,6 +95,106 @@ public void Kill()
             }
         }
 
+        /// <summary>
+        /// Makes a best-effort attempt to kill all descendant (child, grandchild, etc.) processes.
+        /// </summary>
+        private void KillTree()
+        {
+            try
+            {
+                /*
```

Comment:
Same nit as before- we use `\\` for comments, even when they're multiline.

Reply:
Fixed